Letters/src/main/java/org/Marta/UppercaseText.java

```diff
@@ -0,0 +1,8 @@
+package org.Marta;
+
+public class UppercaseText implements TextFormatter{
```

Comment:
analogicznie 
"Jak nie masz pomysłu na nazwę implementacji to pierw spróbuj od rozszerzenia nazwy interfejsu ;)

Interfejs: TextFormatter klasa LowerCaseTextFormatter


"LowercaseText" nie brzmi jak metoda, ktora dostarcza formater "